### ISSUE TITLE
[lightbox-media-viewer - with carousel]: QA: screen reader identifies the hidden slides

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8916]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2517
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Slides in the Carousel

Issue Description:
Total there are 5 slides but only 1 is visible. But screen reader identifies the hidden slides as well. This confuses the user.

Expected:
Screen reader should identify only the visible slides. Hide the slides which are not visible from the screen reader as well.

User Impact:
If the screen reader identifies the visually hidden content, it tends to mislead users who are blind. 

### Component(s) impacted

lightbox-media-viewer - with carousel

### Browser

Safari, Firefox

### Carbon for IBM.com version

canary

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8757/iframe.html?id=components-lightbox-media-viewer--with-carousel

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
